src/plugins/intel_cpu/src/nodes/executors/acl/acl_eltwise.hpp

```diff
@@ -73,28 +75,98 @@ class AclEltwiseExecutorBuilder : public EltwiseExecutorBuilder {
                 return false;
         }
 
-        // ACL supports only U8 precision on output for comparison operations
-        if (one_of(eltwiseAttrs.algorithm, Algorithm::EltwiseEqual, Algorithm::EltwiseNotEqual, Algorithm::EltwiseGreater,
-                                           Algorithm::EltwiseGreaterEqual, Algorithm::EltwiseLess, Algorithm::EltwiseLessEqual)) {
-            if (dstDescs[0]->getPrecision() != InferenceEngine::Precision::U8) {
-                return false;
+        auto checker = [](const std::vector<MemoryDescPtr>& srcDescs,
+                          const std::vector<MemoryDescPtr>& dstDescs,
```

Comment:
you can get access to descriptors by passing them via lambda capture list instead of passing via attributes. Will allow to siplify the call: `checker({Precision::I32, Precision::I32}, Precision::I32)`